wordinserter/renderers/com.py

```diff
@@ -595,6 +595,11 @@ def handle_format(self, op, parent_operation, element_range):
                 if op.text_align in alignment:
                     parent_operation.render.cell_object.Range.ParagraphFormat.Alignment = alignment[op.text_align]
 
+        if op.writing_mode:
+            orientations = {"vertical-lr": 1, "sideways-lr": 2}
+            if isinstance(parent_operation, TableCell):
+                parent_operation.render.cell_object.Range.Orientation = orientations[op.writing_mode]
```

Comment:
Sure. By the way, in my first attempt I used `element_range`, but that resulted in more than one cell being incorrectly vertically-aligned. I wonder if that should work, and if it might be a bug that affects other things.